### ISSUE TITLE
Dashboard graphs configure for better accessibility

### ISSUE THREAD

**Issue by adamjarling** [samvera/hyrax#3040]
### Descriptive summary

From the Accessibility 360 Audit Report:
> "User Activity", "Repository Growth", and "Repository Objects" graphs.
> These graphs are not accessible.
> 
> Add tables to this page which communicate the same information provided by the graphs. Hide these svg elements from screen readers using aria-hidden="true".

![dashboard-graphs](https://user-images.githubusercontent.com/3020266/39712201-cecbe070-51e7-11e8-9c69-80cd640d523c.png)

### Rationale

Accessibility improvement.

### Tasks
- [ ] Add tables to this page which communicate the same information provided by the graphs.
- [ ] Hide these svg elements from screen readers using aria-hidden="true".
